client/scss/settings/_variables.scss

```diff
@@ -70,9 +70,14 @@ $color-button-no: $color-red-dark;
 $color-button-no-hover: darken($color-button-no, 20%);
 $color-button-warning: $color-orange-dark;
 $color-button-warning-hover: darken($color-button-warning, 20%);
+
 $color-link: $color-teal-darker;
 $color-link-hover: $color-teal-dark;
 
+// The focus outline color is defined without reusing a named color variable
+// because it shouldn’t be reused for anything else in the UI.
+$color-focus-outline: #ffbf47;
```

Comment:
This color doesn't pass AA for contrast against white, but it's not text so I know that's not entirely relevant. Does anyone know of a way for assessing contrast of non-text colors?

Reply:
Yes, SC 1.4.11 is specifically about contrast of UI elements, and defines a minimum contrast of 3:1: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html. Unfortunately this shows no notion of the "size" of the elements like 1.4.3 has for text.

We went with this particular shade of yellow because it struck a good compromise between being visible and not being too distracting, after having tried all of the colors in Wagtail’s existing palette, plus custom blues and oranges, against all of the elements in the styleguide and on a few main pages. Our main goal was to have something that stood out against the dark teal used for primary buttons and in the header, and also against the dark grey of the menu. And also a bit of every other shade :)

In the end this was the shade we liked the most in a wide variety of situations, the only place where it sucked was over the "warning" orange/yellow banners used for revisions, and the orange "Save" button also for revisions. These felt like rare enough combinations to not be worth affecting the choice too much.

If you or anyone else wants to have a look at other shades, here is my "outline color switcher" code:

```scss

.outline-teal *:focus { outline-color: $color-teal !important };
.outline-teal-darker *:focus { outline-color: $color-teal-darker !important };
.outline-teal-dark *:focus { outline-color: $color-teal-dark !important };
.outline-new *:focus { outline-color: #ffbf47 !important };
.outline-blue *:focus { outline-color: $color-blue !important };
.outline-red *:focus { outline-color: $color-red !important };
.outline-red-dark *:focus { outline-color: $color-red-dark !important };
.outline-orange *:focus { outline-color: $color-orange !important };
.outline-orange-dark *:focus { outline-color: $color-orange-dark !important };
.outline-green *:focus { outline-color: $color-green !important };
.outline-salmon *:focus { outline-color: $color-salmon !important };
.outline-salmon-light *:focus { outline-color: $color-salmon-light !important };
.outline-white *:focus { outline-color: $color-white !important };
.outline-black *:focus { outline-color: $color-black !important };
```

```js
const select = document.createElement('select');
select.style = `position: fixed; bottom: 0; right: 0; width: 200px;`;

const colors = [
  'outline-new',
  'outline-blue',
  'outline-red-dark',
  'outline-orange',
  'outline-orange-dark',
];

select.addEventListener('change', (e) => {
  colors.forEach(color => {
    document.body.classList.remove(color);
  });
  document.body.classList.add(e.target.value);
});

colors.forEach(color => {
  const option = document.createElement('option');
  option.value = color;
  option.innerHTML = color;
  select.appendChild(option);
});

document.body.appendChild(select);
```